core/src/main/java/org/jboss/set/mjolnir/archive/mail/MailBodyMessageProducer.java

```diff
@@ -0,0 +1,54 @@
+package org.jboss.set.mjolnir.archive.mail;
+
+import j2html.tags.DomContent;
+import org.jboss.set.mjolnir.archive.domain.RemovalStatus;
+import org.jboss.set.mjolnir.archive.domain.UserRemoval;
+
+import javax.inject.Inject;
+import java.util.List;
+
+import static j2html.TagCreator.*;
+import static j2html.TagCreator.td;
+
+public class MailBodyMessageProducer {
```

Comment:
Nice, can you mail me example output, when you have it working? :)

Reply:
Sended